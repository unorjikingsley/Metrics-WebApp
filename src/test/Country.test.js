import React from 'react';
import renderer from 'react-test-renderer';
import Country from '../Components/country';

test('should match the snapshot', () => {
  const home = renderer.create(<Country />);
  const tree = home.toJSON();
  expect(tree).toMatchSnapshot();
});
