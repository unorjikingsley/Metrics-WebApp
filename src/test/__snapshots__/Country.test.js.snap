// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<button
  type="button"
>
  <div
    className="arrow"
  >
    <i
      className="fa-solid fa-circle-arrow-right arrow white"
    />
  </div>
  <div
    className="flags"
  >
    <img
      alt="a country's graphic rep"
      className="graphics"
    />
    <div
      className="details"
    >
      <h2
        className="countryname color"
      />
      <h3
        className="color"
      >
         
        Capital :
         
      </h3>
      <h3
        className="color"
      >
         
        Population :
         
      </h3>
      <h3
        className="color"
      >
         
        Region :
         
      </h3>
    </div>
  </div>
</button>
`;
